WriteHeader/calibration.py

```diff
@@ -0,0 +1,23 @@
+import os
+from numpy import matrix as M
```

Comment:
In Python, it's nice to avoid `from a import b` because the next person to read your code will get confused. It's better to put `import numpy as np` and call `np.matrix` so your code is more readable.